packages/react-native-win32/package.json

```diff
@@ -3,8 +3,8 @@
   "version": "0.0.0-master.3",
   "description": "Implementation of react native on top of Office's Win32 platform.",
   "license": "MIT",
-  "main": "./Libraries/react-native/react-native-implementation.win32.js",
-  "typings": "./Libraries/react-native/typings-main.d.ts",
+  "main": "./index.win32.js",
+  "typings": "./typings-index.d.ts",
```

Comment:
>index [](start = 24, length = 5)

Renamed from `typings-main` to `typings-index` for consistency with RNW. Though I'm not sure why we didn't just go with `index.d.ts`